fsnotify_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2016 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build !plan9,!solaris,!windows
+
+// Package fsnotify provides a platform-independent interface for file system notifications.
+package fsnotify
+
+import (
+	"testing"
+)
+
+func TestOpStringify(t *testing.T) {
+	if Create.String() != "Create" {
+		t.Fatalf("Stringify failed for: Create Op.")
```

Comment:
You can just use t.Error() here, reporting on any errors rather than failing immediately.